### PR TITLE
Create OSMUK-Cadastral-Parcels.geojson

### PR DESCRIPTION
**Pull request by RobJN**
New overlay layer hosted by the OpenStreetMap UK Local Chapter. Hopefully I have followed the schema properly.

### CODE REVIEW

**Comment by RobJN** [comment]
The check failing is due to no privacy policy url added. I didn't add one as the contribution guide said is was "recommended" (not compulsory) and we have no logging on the server so no personal data stored.

File: sources/europe/gb/OSMUK-Cadastral-Parcels.geojson [hunk 1]
```diff
@@ -0,0 +1,1119 @@
...
+    "max_zoom":20,
+    "permission_osm":"implicit",
+    "license_url":"https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "id":"OSMUK-Cadastral-Parcels",
+    "description":"Perectly aligned cadastral (land registry) parcels in Great Britain",
+    "country_code":"GB",
+    "overlay":true,
+    "attribution":{
+      "text":"Contains public sector information licensed under the Open Government Licence v3.0.",
+      "url":"https://www.openrailwaymap.org/"
```
Comment (rbuffat) [inline sources/europe/gb/OSMUK-Cadastral-Parcels.geojson]: Is the openrailwaymap.org intended as url?

**Comment by rbuffat** [comment]
@RobJN Ideally, if there is a policy available it is included. Unfortunately, there is currently no value to denote that there is none. But we can merge the PR even if the test fails.

File: sources/europe/gb/OSMUK-Cadastral-Parcels.geojson (hunk 1 above)
Comment (RobJN) [reply sources/europe/gb/OSMUK-Cadastral-Parcels.geojson]: Oh. I was so careful as well and I still managed to miss something. You can tell which overlay layer I used as the starting point! Now fixed.

**Comment by RobJN** [comment]
I fixed the attribution url and put some basic text on the osmuk.org website to confirm that we have no logging on the server. I linked to that text via the privacy policy url.

File: sources/europe/gb/OSMUK-Cadastral-Parcels.geojson (hunk 1 above)
Comment (rbuffat) [reply sources/europe/gb/OSMUK-Cadastral-Parcels.geojson]: No worries!

**Comment by rbuffat** [comment]
Thanks @RobJN !

**Pull request merged**